wgpu-hal/src/gles/egl.rs

```diff
@@ -532,12 +532,42 @@ impl Inner {
                 .query_string(Some(display), khronos_egl::CLIENT_APIS)
                 .unwrap()
                 .to_string_lossy();
-            client_apis
+            if client_apis
                 .split(' ')
                 .any(|client_api| client_api == "OpenGL")
+            {
+                // check if GL version is supported
+                egl.bind_api(khronos_egl::OPENGL_API).unwrap();
```

Comment:
this goes a bit beyond this PR since technically both the unwrap before and after are affected, but these unwraps make me paranoid! I think we'd better off handling each of them gracefully (more not-supported situations essentially)